renovate.json

```diff
@@ -1,9 +1,24 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
-  "extends": ["config:recommended", "helpers:pinGitHubActionDigests", "helpers:pinGitHubActionDigestsToSemver"],
-  "ignorePaths": ["**/__fixtures__/**", "**/fixtures/**"],
-  "enabledManagers": ["npm", "github-actions", "custom.regex", "devcontainer"],
-  "baseBranches": ["main", "7.17"],
+  "extends": [
+    "config:recommended",
+    "helpers:pinGitHubActionDigests",
+    "helpers:pinGitHubActionDigestsToSemver"
+  ],
+  "ignorePaths": [
+    "**/__fixtures__/**",
+    "**/fixtures/**"
+  ],
+  "enabledManagers": [
+    "npm",
+    "github-actions",
+    "custom.regex",
+    "devcontainer"
+  ],
+  "baseBranches": [
```

Comment:
not related to your PR, but shouldn't we also have `8.x` here?

Reply:
@azasypkin I was torn on this. With the final 8.x release more or less around the corner, I was thinking we'd wait until we had a concrete 8.last branch. I'm not opposed to introducing `8.x` though if we think it'd be useful